kb/0002-scripting-overview.md

```diff
@@ -40,6 +40,461 @@ The following list is sorted alphabetically and is based on the latest build of
 * `program\weather\` - weather system definitions and usage.
 * `program\worldMap\` - everything world map related.
 
+## Typing
+
+In order to be compiled, the scripting files should contain good C-like code. This also means that the variables should be strongly typed. Commonly used are base C types: `int`, `float`, but also `bool` and a simple to use `string`. Any custom structs are supported in the form of `object`s. Last, arrays are supported of the above.
```

Comment:
> Last, arrays are supported of the above.

Is it a correct English sentence?

Reply:
Could be correct but a bit wanky indeed. Will rework.